main.tf

```diff
@@ -208,7 +218,6 @@ resource "aws_ecs_service" "service" {
     for_each = var.service_registry_arn == "" ? [] : [1]
     content {
       registry_arn   = var.service_registry_arn
-      container_port = var.task_container_port
```

Comment:
Please remove this line and rename the PR. Please open a separate PR if #42 doesn't solve your issue.